pixi.toml

```diff
@@ -54,8 +54,7 @@ test = "pytest tests/integration --backend=postgres"
 coverage = "pytest tests/integration --cov=datajudge --cov-report=xml --cov-append --backend=postgres"
 
 [feature.db2.dependencies]
-# Upper bounded due to https://github.com/ibmdb/python-ibmdb/issues/986
-ibm_db = "<=3.2.4"
+ibm_db = "*"
```

Comment:
Shouldn't it be better to scope the misbehaving version(s) out?

Reply:
You're right of course!
[9022938](https://github.com/Quantco/datajudge/pull/269/commits/90229385ee7d68bab3d6e3772a119ebbcd27cdb9)